docs/site/src/demos/lists/InsetList.js

```diff
@@ -0,0 +1,37 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import {
+  List,
+  ListItem,
+  ListItemText,
+} from 'material-ui/List';
+
+const styleSheet = createStyleSheet('InsetList', (theme) => ({
+  root: {
+    width: '100%',
+    maxWidth: '360px',
+    background: theme.palette.background.paper,
+  },
+}));
+
+function SimpleList(props, context) {
+  const classes = context.styleManager.render(styleSheet);
+  return (
+    <List className={classes.root}>
+      <ListItem button>
+        <ListItemText inset primary="Chelsea Otakan" />
```

Comment:
I wanted to do that but there is a margin bug with icon on left.
The padding-left is not good.